package.xml

```diff
@@ -20,15 +20,18 @@
   <author>William Woodall</author>
   <author>Robert Haschke</author>
 
-  <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>ros2launch</build_depend>
```

Comment:
As far as I can tell, *this* PR doesn't do anything with launch.  So we don't need this.